code/modules/paperwork/paper.dm

```diff
@@ -5,6 +5,15 @@
  * lipstick wiping is in code/game/objects/items/weapons/cosmetics.dm!
  */
 
+#define PAPER_FIELD "<span class=\"paper_field\"></span>"
```

Comment:
Since this define is only used in this file, undef it at the end of the file

Reply:
BITCH I DID

Reply:
undef it harder